core/kubernetes/config/config-importer-job.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: core-config-importer
+  name: openhim-config-importer-job
```

Comment:
I think the intention here with using `core` was in reference to the `Core Package` as this config script could in theory also handle the HAPI FHIR import if needed

Reply:
@BMartinos I see that, however, I think that by including the potential for including HAPI-FHIR in this import script would be mixing up unrelated concerns.
The majority of this job is focused on openhim specifics. Adding HAPI-FHIR content here doesn't seem right. 
We also discussed a few weeks back how we would have separate scripts for unrelated imports and we would set up our openhim-config import script as an example of how to do this - instead of stringing together commands in one container. This would also help implementors debug import issues. And enable us to run specific imports without changing any code.

Reply:
ok cool that makes sense. Lets keep this as is for now then. We might just have to rethink the naming convention in future if in-case we have another openhim instance being spun up in a different package :+1: